dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/UnitOfWorkAwareProxyFactory.java

```diff
@@ -100,8 +100,9 @@ public UnitOfWorkAwareProxyFactory(HibernateBundle<?>... bundles) {
 
     /**
      * @return a new
+     * @param sessionFactories
      */
-    public UnitOfWorkAspect newAspect() {
-        return new UnitOfWorkAspect(sessionFactories);
+    public UnitOfWorkAspect newAspect(ImmutableMap<String, SessionFactory> sessionFactories) {
+        return new UnitOfWorkAspect(this.sessionFactories);
```

Comment:
Yes, that was incorrect. Fixed and also made it an overload so we could keep the no args version.